src/schibsted/payment/sdk/adapters/Curl.php

```diff
@@ -18,9 +18,9 @@ class Curl extends \schibsted\payment\lib\sdk\Adapter
     /**
      * Makes an HTTP request.
      *
-     * @param String $uri the URI to make the request to
-     * @param Array $post the parameters to use for the POST body
-     * @param CurlHandler $ch optional initialized curl handle
+     * @param string $uri the URI to make the request to
+     * @param array $post the parameters to use for the POST body
+     * @param \GuzzleHttp\Ring\Client\CurlHandler $ch optional initialized curl handle
```

Comment:
This is an actual Curl adapter that is not related to Guzzle. So this is an invalid change

Reply:
Can you please point me to the actual file where the adapter is defined?

Reply:
THIS file.. the one you changed and we are commenting on :)

Reply:
Sorry, my bad. I was confused by the fact that CurlHandler class does not exist. If it existed then it'd be a violation of PSR-0 autoloading.  

By your comment, the parameter should be changed to `\schibsted\payment\sdk\adapters\Curl`. However, the thing is that the the parameter will actually be a resource (check `curl_init` return), so the type-hint against anything except resource would be misleading. As this is just a part of the care-requiring codebase, I'll revert my changes, to not open a Pandora box. :)

Reply:
The value that is returned does not have a class, it is type hinted only as "resource" in the php docs : http://php.net/manual/en/function.curl-init.php so I am not sure what typehint would would here. That docblock line was written like that for human to read not IDE I guess.